src/Components/TodoList/TodoList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ToDoItem from '../ToDoItem/ToDoItem';
+import './TodoList.scss';
+
+const TodoList = ({ preparedToDos }) => (
+  <section className="main__container">
+    {preparedToDos.map(todo => (
+      <ToDoItem todo={todo} key={todo.id} />
+    ))}
+  </section>
+);
+
+TodoList.propTypes = {
+  preparedToDos: PropTypes.arrayOf(PropTypes.shape({}).isRequired).isRequired,
```

Comment:
describe shape

Reply:
added 'id' field for validation in TodoList component. The rest of the fields of are validated in ToDoItem component.